front-end/src/components/FriendRequestButton.tsx

```diff
@@ -53,7 +54,7 @@ export default function FollowRequestButton(props: Props) {
 
   return (
     <>
-      <Button className="list-group-item" color="link" onClick={sendFollowRequest}>{props.isFollower ? "Unfollow" : "Follow"}</Button>
+      <Button style={{outline: 'none'}} color="white" onClick={sendFollowRequest}>{props.isFollower ? Icons.unfollowIcon : Icons.followIcon}</Button>
```

Comment:
Nit: As a fun fact, my mentor told me in production one should never set `outline` to `none`, since it affects accessibility (makes it harder to navigate via keyboard).
https://stackoverflow.com/questions/19455499/why-is-outlinenone-not-considered-good-practice

This is a fun fact only, I'm more than fine leaving it in for our usecase

Reply:
I'll change this. what color should the button be? might look a lil wonky which is my concern

Reply:
Your pick! I only brought it up since you seem fairly invested in FE stuff, so I'd like to pass on anything I know. If it looks wonky we can leave it as is

Reply:
I took out the style/outline stuff, doesn't seem to have changed anything so we're good